src/test/java/io/managed/services/test/rhoas/KafkaRhoasBasicTests.java

```diff
@@ -517,6 +526,35 @@ public void testDeleteServiceAccount() {
             () -> cli.describeServiceAccount(serviceAccount.getClientId()));
     }
 
+    @Test(dependsOnMethods = "testLogin", enabled = true)
```

Comment:
consider logic of the test, we don't need to test create kafka instance twice 
(there is a test which called apply kafka instance, which to be honest create kafka instance, apply part is there only because for debug part).  

this can be probably just a single test `changeKafkaInstanceOwner`, which would be in try catch finaly with its teardown. Proposed solution is due to fact that you do not need this instance for anything else, this solution would create it  (probably as one of first test being executed, due to testng priorities) and keep it doing nothing, while taking space. 

this would be pretty clean in my opinion, other possible solution would be to reuse kafka instance you are using in other tests in this class, but that would mean ordering it correctly and definitelly complicating logic of teardown method.

Reply:
Fixed in: [302130a](https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/pull/524/commits/302130a26afec1022546fc27a5fca666652cc2c5) 

Sorry, I didn't see that the apply was already doing a create! Thanls for the comment =)